docs/encoding.md

```diff
@@ -81,7 +82,7 @@ For each of these encodings, prism provides a function for checking if the subse
 
 ## Support for other encodings
 
-If an encoding is encountered that is not supported by prism, prism will call a user-provided callback function with the name of the encoding if one is provided. That function can be registered with `pm_parser_register_encoding_decode_callback`. The user-provided callback function can then provide a pointer to an encoding struct that contains the requisite functions that prism will use those to parse identifiers going forward.
+If an encoding is encountered that is not supported by prism, prism will call an user-provided callback function with the name of the encoding if one is provided. That function can be registered with `pm_parser_register_encoding_decode_callback`. The user-provided callback function can then provide a pointer to an encoding struct that contains the requisite functions that prism will use to parse identifiers going forward.
```

Comment:
Getting rid of `those` is fine, but it should be `a` not `an`.

Reply:
oops you're right, fixed!